qa/run-standalone.sh

```diff
@@ -24,6 +24,15 @@ else
     COREPATTERN="core.%e.%p.%t"
 fi
 
+function finish() {
+    if [ -n "$precore" ]; then
+        sudo sysctl -w ${KERNCORE}=${precore}
+    fi
+    exit 0
```

Comment:
@tchaikov With daemons running if I don't exit here weird things happened.  I don't remember exactly what happened, but I tried it with a common function.  I think I had to ^C twice to get the test to stop running.